tests/fixtures/lxd-test.ts

```diff
@@ -0,0 +1,23 @@
+import { test as base } from "@playwright/test";
+
+export type LxdVersions = "5.0-stable" | "latest-edge";
+const LXD_CHANNEL = process.env.LXD_CHANNEL as LxdVersions;
+
+export const test = base.extend<{
+  lxdVersion: LxdVersions;
+  addLxdVersionAnnotation: null;
+}>({
+  lxdVersion: LXD_CHANNEL,
+  addLxdVersionAnnotation: [
+    // eslint-disable-next-line no-empty-pattern
```

Comment:
This is a bit ugly, maybe there is a better way?

Reply:
have adjusted

Reply:
In playwright's [Typescript examples](https://playwright.dev/docs/test-fixtures#automatic-fixtures) they have empty objects in the first argument for those functions. Not sure how to overcome this one without disabling the linting rule.